CONTRIBUTING.md

```diff
@@ -1,17 +1,19 @@
 Submitting Issues
 -----------------
+
 When submitting issues to our
 [issue tracker](https://github.com/sopel-irc/sopel/issues), it's important
 that you do the following:
 
 1. Describe your issue clearly and concisely.
-2. Give Sopel the .version command, and include the output in your issue.
+2. Give Sopel the `.version` command, and include the output in your issue.
 3. Note the OS you're running Sopel on, and how you installed Sopel (via your
-package manager, pip, setup.py install, or running straight from source)
-4. Include relevant output from the log files in ~/.sopel/logs.
+   package manager, pip, `setup.py install`, or running straight from source)
+4. Include relevant output from the log files in `~/.sopel/logs.`
```

Comment:
IIRC, logs is a folder, and as such, I feel like it's better to add a trailing /

```suggestion
4. Include relevant output from the log files in `~/.sopel/logs/`.
```

I assume the trailing dot wasn't supposed to be included.

Reply:
Indeed, I think that was a regex replace.